Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -287,7 +287,8 @@
 
     <!-- Debug -->
     <string name="debug_submenu">Debug</string>
-    <string name="debug_warning">Warning: These settings will slow emulation</string>
+    <string name="debug_warning">Warning: Debug settings will slow emulation</string>
```

Comment:
Hmm... In the future, we should probably add some kind of way to show messages that don't look identical to headers, but I guess this is okay for now.